src/device-registry/.gitignore

```diff
@@ -2,4 +2,8 @@ node_modules
 controllers/rsa_private.pem
 .env
 device-reg-secret.yaml
-log
\ No newline at end of file
+log
+scripts/data-airqlouds.js
```

Comment:
just ignoring the local files which contain the data for carrrying out bulk operations